src/apache/scripts/httpd-wrapper

```diff
@@ -20,6 +20,13 @@ else
 	echo "No certificates are active: using HTTP only"
 fi
 
+# If no listen address is specified, listen on all IPv4 addresses (0.0.0.0)
+if [ $(apache_listen_address) ]; then
+    export LISTEN_ADDRESS="$(apache_listen_address)"
+else
+    export LISTEN_ADDRESS="0.0.0.0"
+fi
```

Comment:
Please follow the pattern created by the port configuration. Have the default in the Apache utilities instead of putting the logic here.